.github/maintainers_guide.md

```diff
@@ -64,7 +66,7 @@ Releasing can feel intimidating at first, but rest assured: if you make a mistak
 
 6. Once the PR has been approved and tests have passed, merge it into the main repository.
```

Comment:
Discussion point here: should we keep in this line about PR approvals? Fil made a great point that since releases are a pretty straightforward change of the versioning, they typically don't require any reviewing. On one hand, having this line here would create another level of confidence for the version bump - on the other hand, it may be a blocker for someone releasing to wait for an approval. Would love to get thoughts here!

Reply:
Keeping this in does requiring involving 2 people for each release I guess this approval ensures that the "versioning" PR remains minor and insignificant. At the same time if we all are on the same page this approval can be removed to facilitate the release process.

Reply:
Personal opinion: I think we can remove the approval for these PRs, but let me ping the team to field more opinions.

Reply:
updating with the general consensus for this: 
- main concern with completely removing approval phrasing is not having a secondary check on the manual change in `package.json`. however, most folks that review typically will just look at the checks anyways to determine if the PR should be approved or not, not necessarily the diff
- to move forward - we will make the approval step optional in the docs and encourage people to get a review if they'd like a one or if they feel that this specific version bump it needs more attention. we can also make a note in the docs for folks to make sure to wait for the checks pass before merging! i'll throw in an update for this tomorrow morning in this PR! 🙌